api/invite.go

```diff
@@ -36,20 +36,23 @@ func (a *API) Invite(w http.ResponseWriter, r *http.Request) error {
 	if err != nil && !models.IsNotFoundError(err) {
 		return internalServerError("Database error finding user").WithInternalError(err)
 	}
-	if user != nil {
-		return unprocessableEntityError(DuplicateEmailMsg)
-	}
 
 	err = a.db.Transaction(func(tx *storage.Connection) error {
-		signupParams := SignupParams{
-			Email:    params.Email,
-			Data:     params.Data,
-			Aud:      aud,
-			Provider: "email",
-		}
-		user, err = a.signupNewUser(ctx, tx, &signupParams)
-		if err != nil {
-			return err
+		if user != nil {
+			if user.IsConfirmed() {
+				return unprocessableEntityError(DuplicateEmailMsg)
+			}
+		} else {
+			signupParams := SignupParams{
+				Email:    params.Email,
+				Data:     params.Data,
+				Aud:      aud,
+				Provider: "email",
+			}
+			user, err = a.signupNewUser(ctx, tx, &signupParams)
+			if err != nil {
+				return err
+			}
```

Comment:
Bundled fix for #134 as well for convenience. If a user has been signed up but not confirmed yet, an admin should still be able to send out invite links to the same user.